extide/gradle/arch.xml

```diff
@@ -150,6 +150,12 @@
          can be attached to that port, make the <code>netbeans-gradle-tooling</code>
          project debuggable in action.
      </api>
+     <api category="friend" group="property" name="runJvmArgs" type="export">
+         NetBeans Tooling plugin recognizes <code>runJvmArgs</code> project property, and injects
+         specified JVM arguments into JavaExec executing the user code. Use <code>-PrunJvmArgs=...</code>.
+         If passing more arguments, the whole -P must be properly quoted. Arguments that contain spaces must
```

Comment:
Example? I am not sure I can imagine how exactly one would do it, yet the wording seems scary enough.

Reply:
Hm. Only inlined example. In this case, I have no reasonable symbol to hook to with `@codesnippet` ;) 
But maybe just a wrong wording. OK - the plugin just captures `-PrunJvmArgs=...` property from `gradle` commandline; nothing more.